samples/e2eTestUtils/src/TestUtils.ts

```diff
@@ -307,7 +307,22 @@ export async function enterCredentials(
             throw e;
         });
     } catch (e) {
-        return;
+        // keep me signed in MSA
+        try {
+            await page.waitForSelector("#kmsiTitle", { timeout: 1000 });
```

Comment:
Could be a bit faster and less duplication if you use Promise.any above with both tags

Reply:
Great idea, thanks! I was wondering if there was a way to use whichever one showed up.

Reply:
@tnorling ended up using race since .any wasn't available for the current build target settings.